gcp/service-account-key/README.md

```diff
@@ -0,0 +1,11 @@
+## Required environment variables:
```

Comment:
Can you please add some background on what this code is doing and why so it would be clear to whoever stumble upon this.

Reply:
54ef73f